lib/helpers/add-many.js

```diff
@@ -0,0 +1,38 @@
+var utils = require('pouchdb-utils')
```

Comment:
`pouchdb-utils` is missing in the dependencies. 

This may also be the related to why [this test](https://github.com/hoodiehq/hoodie-store-client/blob/hoodiehq/camp%23121/tests/integration/reset.js#L52) is failing for me, because `require('pouchdb-utils').extend` is undefined (it seems pouchdb-utils doesn't export this method anymore https://github.com/pouchdb/pouchdb/releases/tag/6.1.1). Using `pouchdb-utils@5` does work though.